public/wiki-edit-redirect.php

```diff
@@ -18,7 +18,7 @@
     'docs.retroachievements.org' => 'docs',
     'guides.retroachievements.org' => 'guides',
 ];
-$repo = $repoDomainMap[parse_url($_SERVER['HTTP_REFERER'])['host']] ?? null;
+$repo = $repoDomainMap[parse_url($_SERVER['HTTP_REFERER'] ?? null)['host'] ?? null] ?? null;
```

Comment:
Won't this just cause a "cannot index ['host'] from null" error?

Reply:
By using the null-coalescing `??` operator it shouldn't run into an undefined index

Reply:
It does however trigger a deprecation warning - goal wasn't to fully fix it but to prevent it from filling up the logs. A better solution will be introduced when using the controller action from the v2 prototype.

Reply:
I didn't realize `??` short-circuited the evaluation.